wgsl/index.bs

```diff
@@ -4190,6 +4197,46 @@ Example:  `let minint = -2147483648;` is analyzed as follows:
     [=AbstractInt=] -2147483648 value is converted to the [=i32=] value -2147483648.
 * The result is that `minint` is declared to be the i32 value -2147483648.
 
+### `override` Expressions ### {#override-expr}
+
+Expressions that are evaluated at [=pipeline creation=] time are called <dfn
+noexport>override-expressions</dfn>.
+In order for an expression to be evaluated at pipeline creation time all
+[=identifiers=] used by the expression must [=resolve=] to:
+* [=override-declarations=], or
+* [=const-declarations=], or
+* [=const-functions=], or 
+* [=type aliases=], or 
+* [=structure=] names
+
+Override-expressions are a superset of [=const-expressions=], except that they
+cannot [=type rules|resolve=] to an [=abstract type=].
+
+Note: Override-expressions may have a type that is not assignable to an
+[=override-declaration=] because they can resolve to [=scalar|non-scalar=]
```

Comment:
Change to singular subject.

> Note: An override-expression may not be usable as the initializer for an override-declaration, because such initializers must be a scalar or feasibly convertible to a scalar.  But override-expressions can be of composite type.